app/screens/conversion-flow/use-convert-money-details.ts

```diff
@@ -0,0 +1,95 @@
+import { DisplayCurrency, MoneyAmount, WalletOrDisplayCurrency } from "@app/types/amounts"
+import React from "react"
+import { usePriceConversion } from "../../hooks/use-price-conversion"
+import { WalletCurrency } from "@app/graphql/generated"
+
+type Wallet = {
+  balance: number
+  walletCurrency: WalletCurrency
+  id: string
+}
+
+export const useConvertMoneyDetails = () => {
+  const [wallets, _setWallets] = React.useState<{
+    fromWallet: Wallet
+    toWallet: Wallet
+  }>()
```

Comment:
shouldn't this have a default?

Reply:
It gets set right away as soon as the component calling the hook knows the wallets

Reply:
I can add an initial argument that will try to set it, but it will still be potentially undefined.